examples/shapes/shapes_recursive_tree.c

```diff
@@ -0,0 +1,138 @@
+*
+*   Example complexity rating: [★★★☆] 3/4
+*
+*   Example originally created with raylib 5.6-dev, last time updated with raylib 5.6-dev
+*
+*   Example contributed by Jopestpe (@jopestpe)
+*
+*   Example licensed under an unmodified zlib/libpng license, which is an OSI-certified,
+*   BSD-like license that allows static linking with closed source software
+*
+*   Copyright (c) 2018-2025 Jopestpe (@jopestpe)
+*
+********************************************************************************************/
+
+#include "raylib.h"
+#include <math.h>
+
+#define RAYGUI_IMPLEMENTATION
+#include "raygui.h"                 // Required for GUI controls
+
+//----------------------------------------------------------------------------------
+// Types and Structures Definition
+//----------------------------------------------------------------------------------
+typedef struct {
+    Vector2 start;
+    Vector2 end;
+    float angle;
+    float length;
+} Branch;
+
+//----------------------------------------------------------------------------------
+// Module Functions Declaration
+//----------------------------------------------------------------------------------
+static Vector2 CalculateBranchEnd(Vector2 start, float angle, float length);
+
+//------------------------------------------------------------------------------------
+// Program main entry point
+//------------------------------------------------------------------------------------
+int main(void)
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [shapes] example - shapes recursive tree");
+
+    Vector2 start = { (screenWidth / 2.0f) - 125, screenHeight };
+    float angle = 40.0f;
+    float thick = 1.0f;   
+    float treeDepth = 10.0f;
+    float branchDecay = 0.66f;
+    float length = 120.0f;
+    bool bezier = false;
+
+    SetTargetFPS(60);               // Set our game to run at 60 frames-per-second
+    //--------------------------------------------------------------------------------------
+
+    // Main game loop
+    while (!WindowShouldClose())    // Detect window close button or ESC key
+    {
+        // Update
+        //----------------------------------------------------------------------------------
+        float theta = angle * DEG2RAD;
+        int maxBranches = (int)(powf(2, (int)(treeDepth))); 
+        Branch branches[4096];
+        int count = 0;
+
+        Vector2 initialEnd = CalculateBranchEnd(start, 0.0f, length);
+        branches[count++] = (Branch){start, initialEnd, 0.0f, length};
+
+        for (int i = 0; i < count; i++) {
+            Branch branch = branches[i];
+            if (branch.length < 2) continue;
+
+            float nextLength = branch.length * branchDecay;
+
+            if (count < maxBranches && nextLength >= 2) {
```

Comment:
Please, could you review it to follow raylib code conventions with aligned brackets?